research-hub-web/cypress/integration/basic-functionality.e2e.js

```diff
@@ -9,12 +9,12 @@ describe('ResearchHubs Basic Functionality', () => {
     });
 
     it('has title', () => {
-        cy.contains('Welcome to the ResearchHub');
+        cy.contains('Welcome to ResearchHub');
```

Comment:
[Steady on there Steve Parker](https://www.youtube.com/watch?v=PEgk2v6KntY), I think we'll stick with _the ResearchHub_ for now